test/AspNet.Security.OAuth.Providers.Tests/Kloudless/KloudlessTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AspNet.Security.OAuth.Kloudless
+{
+    public class KloudlessTests : OAuthTests<KloudlessAuthenticationOptions>
+    {
+        public KloudlessTests(ITestOutputHelper outputHelper)
+        {
+            OutputHelper = outputHelper;
+        }
+
+        public override string DefaultScheme => KloudlessAuthenticationDefaults.AuthenticationScheme;
+
+        protected internal override void RegisterAuthentication(AuthenticationBuilder builder)
+        {
+            builder.AddKloudless(options => ConfigureDefaults(builder, options));
+        }
+
+        [Theory]
+        [InlineData(ClaimTypes.NameIdentifier, "my-id")]
```

Comment:
Could you extend the test to cover all of the custom claim mappings please?

Reply:
I've added the tests for as much mapping as I could, but as `AuthenticateUserAsync` returns a Dictionary, I can't add mapping test for array (apis property)